### PR TITLE
Function Names

### PR DESCRIPTION
**Pull request by Octo-git**
Added function names, Issue:92 entire src directory, does not include the __test__ directory

### CODE REVIEW

**Comment by Octo-git** [comment]
Note: I did not change scheduler.js because another user already stated they will be working on it.

**Comment by ILOVEPIE** [comment]
> Note: I did not change scheduler.js because another user already stated they will be working on it.

I'll probably end up doing that file myself as they did it wrong and I haven't heard back regarding correcting it. Thank you for reading the directions, sorry about the amount of function renaming, we recently found out that debuggers were struggling to get the correct function names if we didn't do this. These changes should make debugging and performance analysis a lot easier.

**Comment by ILOVEPIE** [comment]
I'll review it when I wake up. It's 3am where I am and I need to sleep.

**Comment by Octo-git** [comment]
> > Note: I did not change scheduler.js because another user already stated they will be working on it.
> 
> I'll probably end up doing that file myself as they did it wrong and I haven't heard back regarding correcting it. Thank you for reading the directions, sorry about the amount of function renaming, we recently found out that debuggers were struggling to get the correct function names if we didn't do this. These changes should make debugging and performance analysis a lot easier.

No prob, I just reused some regex match strings for all the files with some eyeballing, it didn't a long time

File: src/sabre.js
Comment (ILOVEPIE) [inline src/sabre.js:1]: Somehow you indented this file by one space, can you amend your commit and exclude this change.

**Review by ILOVEPIE — CHANGES_REQUESTED** [review]
There's one small change that needs to be made to your commit by amending it, the sabre.js file got indented somehow, that change needs to be removed from the commit. If you don't know how to do that I can walk you through it. Let me know if you need help.

**Comment by ILOVEPIE** [comment]
why did you close this.

**Comment by ILOVEPIE** [comment]
all you have to do is amend the commit, that doesn't require deleting the repository.

**Pull request closed without merging**